packages/application-shell/src/components/project-switcher/project-switcher-test-utils.ts

```diff
@@ -1,13 +1,19 @@
 import { GRAPHQL_TARGETS } from '@commercetools-frontend/constants';
 import ProjectsQuery from './project-switcher.mc.graphql';
 
+type CreateGraphqlResponseForProjectsQueryOptions = {
```

Comment:
Nitpick: I'm not deeply familiar with the AppKit, but if the preceding `T` for types like done a previous change (from `Props` to `TProps`) is a convention that u have here, should we keep it here?

Reply:
Usually we prefix the types with `T` when the types are exported and used somewhere else. For types that are only used within the module itself, it's not a must.
We also don't have a strict convention but that's more or less what we've been doing so far.

Reply:
I actually like having all types start with T. Internal to file or not. Maybe just me 😄